dataland-frontend/tests/e2e/support/commands.ts

```diff
@@ -87,6 +88,16 @@ export function logout(): Chainable<JQuery> {
         .should("be.visible")
 }
 
+export function logoutDropdown() : Chainable<JQuery> {
```

Comment:
Mich würde dein Gedanke interessieren, weshalb du den Test in eine Funktion gepackt hast.